exporters/otlp/options.go

```diff
@@ -155,3 +156,55 @@ func WithGRPCDialOption(opts ...grpc.DialOption) ExporterOption {
 		cfg.grpcDialOptions = opts
 	}
 }
+
+// ConnConfigurations keeping track of both traces and metrics
+type ConnConfigurations struct {
+	metrics config
+	traces  config
+}
+
+// DefaultConnectionOptions gives a simple way to configure the Exporter connections to default.
+var DefaultConnectionOptions []ExporterOption = []ExporterOption{
+	WithAddress(fmt.Sprintf("%s:%d", DefaultCollectorHost, DefaultCollectorPort)),
+	WorkerCount(DefaultNumWorkers),
+	WithGRPCServiceConfig(DefaultGRPCServiceConfig),
+}
+
+// NewConnections creates an empty configuration and applies
+// any ExporterOptions provided for both metrics and traces.
+func NewConnections(opts ...ExporterOption) ConnConfigurations {
+	return ConnConfigurations{
+		metrics: applyOptions(config{}, opts...),
+		traces:  applyOptions(config{}, opts...),
+	}
+}
+
+// SetCommonOptions updates the connection configuration for both metrics and traces,
+// allowing to override default or custom options for both endpoints at once.
+func (occ ConnConfigurations) SetCommonOptions(opts ...ExporterOption) ConnConfigurations {
```

Comment:
Another possibly useful method allowing client to override options for both signals at once. The main use case would be:
```go
otlp.NewConnections(otlp.DefaultConnectionOptions...).SetCommonOptions(commonOpts...)
```